composer.json

```diff
@@ -48,6 +48,10 @@
     "suggest": {
         "ext-pdo_sqlite": "For running functional tests out-of-the-box this is required"
     },
+    "allow-plugins": {
+        "neos/composer-plugin": true,
+        "composer/package-versions-deprecated": true
+    },
```

Comment:
This needs to be inside the `config` section.

Reply:
Yap that is correct